internal/ui/settings_ui.go

```diff
@@ -378,6 +378,25 @@ func getPicturesDir() (fyne.ListableURI, error) {
 		return nil, err
 	}
 
-	uri := storage.NewFileURI(filepath.Join(home, "Pictures"))
-	return storage.ListerForURI(uri)
+	if runtime.GOOS == "darwin" {
+		uri, err := storage.Child(storage.NewFileURI(home), "Pictures")
+		if err != nil {
+			return nil, err
+		}
+
+		return storage.ListerForURI(uri)
+	}
+
+	const xdguserdir, pic = "xdg-user-dir", "PICTURES"
+	cmd := exec.Command(xdguserdir, pic)
+	stdout := bytes.NewBufferString("")
```

Comment:
You can simplify this by using `Command.Output()` I think

Reply:
I realise now you would have got this from the other XDG code in Fyne - that probably needs tidied up further!

Reply:
I noticed that you did so. Thank you :+1: